### DISCUSSION TITLE
Welcome to kaiaulu Discussions!

### DISCUSSION THREAD

**Discussion by carlosparadis** [sailuh/kaiaulu#87] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CorneJB** [comment]
Hiya! My name is Corné Broere and I'm a master's student from the Netherlands, studying digital business and innovation. I'm hoping to utilize kaiaulu to conduct my thesis research on the possible effects of OSS license choice on the community smells that occur in a community. During my use of kaiaulu I hope I can contribute to the community through the viewpoint of a user, and maybe after some experience contribute to the tool itself :)
